js/src/collapse.js

```diff
@@ -363,7 +363,9 @@ const Collapse = (($) => {
    */
 
   $(document).on(Event.CLICK_DATA_API, Selector.DATA_TOGGLE, function (event) {
-    event.preventDefault()
+    if (!$(event.target).is('input')) {
```

Comment:
Yes thank you @vanduynslagerp ! That's the regex I looked for but didn't found it 😄